source/stylesheets/_landing.scss

```diff
@@ -196,6 +208,19 @@
   input[type="submit"] {
     @include button-callout($secondary-button-color);
   }
+
+  .text {
+    @include media($medium-screen) {
+      @include span-columns(4 of 10);
+    }
+  }
+
+  #mc_embed_signup {
```

Comment:
Avoid using id selectors<br>Selector `mc_embed_signup` should be written in lowercase with hyphens